### PR TITLE
Default shiftKey behavior on transformer

### PR DESCRIPTION
**Pull request by pro-nav**

### CODE REVIEW

**Comment by pro-nav** [comment]
https://github.com/konvajs/konva/issues/1453#issue-1453071723

File: src/shapes/Transformer.ts [hunk 1]
```diff
@@ -1570,6 +1583,21 @@ Factory.addGetterSetter(Transformer, 'borderDash');
...
+ * @param {String} shiftBehavior
+ * @returns {String}
+ * @example
+ * // get
+ * var shiftBehavior = transformer.shiftBehavior();
+ *
+ * // set
+ * transformer.shiftBehavior("none");
+ */
+Factory.addGetterSetter(Transformer, 'shiftBehavior', "default");
```
Comment (Pi-Bouf) [inline src/shapes/Transformer.ts:1599]: Keep single quote on the `default` value !

**Review by Pi-Bouf — CHANGES_REQUESTED** [review]

**Review by Pi-Bouf — APPROVED** [review]

**Review by MrHeer — APPROVED** [review]
LGTM

**Comment by lavrton** [comment]
Please fix tests. I am ok to merge it.

**Comment by lavrton** [comment]
Merged in https://github.com/konvajs/konva/pull/1535

**Pull request closed without merging**